matter/efr32/efr32mg12/BRD4161A/autogen/sl_event_handler.c

```diff
@@ -49,6 +50,7 @@ void sl_kernel_start(void)
 void sl_driver_init(void)
 {
     GPIOINT_Init();
+    sl_spidrv_init_instances();
```

Comment:
Thread example do not use the spi drv instances for the expansion board. It might be preferable to wrap this new call in a #ifdef SL_WIFI.

This and the include at line 23. This would apply to all boards you added this to.